ADAL/src/cache/ios/ADKeychainTokenCache.m

```diff
@@ -35,12 +35,19 @@
 #define KEYCHAIN_VERSION 1
 #define STRINGIFY(x) #x
 #define TOSTRING(x) STRINGIFY(x)
+#define ONE_DAY_IN_SECONDS 86400
 
 static NSString* const s_nilKey = @"CC3513A0-0E69-4B4D-97FC-DFB6C91EE132";//A special attribute to write, instead of nil/empty one.
 static NSString* const s_delimiter = @"|";
 
 static NSString* const s_libraryString = @"MSOpenTech.ADAL." TOSTRING(KEYCHAIN_VERSION);
 
+// artificial key made up for storing the next tombstone clean time in keychain
```

Comment:
Don't store this as an artificial token, that'll trip up old versions and confuse developers that are going through the allItems array. This should be a separate keychain entry independent of the tokens themselves.